test/test_acceptance.py

```diff
@@ -59,7 +62,11 @@ def test_given_veg_case_runs(self):
             hydrodynamics=dict(
                 working_dir=test_dir / "d3d_work",
                 d3d_home=kernels_dir,
-                dll_path=kernels_dir / "dflowfm_with_shared" / "bin" / "dflowfm.dll",
+                dll_path=kernels_dir
+                / "dflowfm_with_shared"
+                / "bin"
+                / "dflowfm"
```

Comment:
I'm not sure why this has changed from the previous cases. The dll path should always be the same for all use cases.
Because of the way BMI works we recommend creating a 'dflowfm_with_shared'direcotry, inside it a bin folder with all the dll's inside, no extra directories (as described in the changed code)